day2/question1/solution1.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+var fre map[string]int
+var wg sync.WaitGroup
+
+func countFrequency(s string) {
+	l := len(s)
+	for i := 0; i < l; i++ {
+		fre[string(s[i])]++
```

Comment:
concurrency is not handled

Reply:
defining another Add() goRoutine  would fix that

Reply:
Done by spawning IncrementByOne() goRoutine